java/fury-benchmark/src/main/java/io/fury/benchmark/Identity2IdMap.java

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2014 Ruediger Moeller.
```

Comment:
This is a fiel coming from https://github.com/RuedigerMoeller/fast-serialization for map benchmarks only. Maybe we should reserver it's heade?

Reply:
Yes. Updating...